### DISCUSSION TITLE
Shorter token without login

### DISCUSSION THREAD

**Discussion by WingLim** [djyde/cusdis#150] [Q&A]
Hi, I made another Cusdis telegram bot which can approve and append comment in telegram.

The project in here [cusdis-telegram-bot](https://github.com/WingLim/cusdis-telegram-bot)

For now I am using a serverless redis to store approve link.

Because telegram bot inline button's [callback_data](https://core.telegram.org/bots/api#inlinekeyboardbutton) can only support 64bytes data maximum.

So I wonder if the token can be shorter that the bot no need to use a redis and can be handled in telegram.

**Comment by djyde** [comment]
Make sense to open an option for enable short token.
